packages/amplify-category-hosting/lib/category-manager.js

```diff
@@ -74,8 +76,54 @@ async function migrate(context) {
   await sequential(migrationTasks);
 }
 
+
+function getIAMPolicies(resourceName, crudOptions) {
+  let policy = {};
+  let actions = new Set();
+
+  crudOptions.forEach((crudOption) => {
+    switch (crudOption) {
```

Comment:
In the storage category we have 'create/update' option instead of both.  I think it makes sense because it's illustrative of what's actually happening for the user, even though it's at the expense of slight inconsistency.   Not a blocker for merging this but something that I think should be considered.